### PR TITLE
ci: housekeeping

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,22 +14,14 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: 1.13
-              otp: 24.3
+              elixir: 1.13.x
+              otp: 22.x
```
Comment (kianmeng) [inline .github/workflows/ci.yml:18]: Minimum support OTP for Elixir 1.13.x is OTP 22, see https://hexdocs.pm/elixir/1.13/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp

**Review by isimluk — APPROVED** [review]
:+1:

**Comment by zachallaun** [comment]
Thanks for the PRs!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**